include/swift/AST/DiagnosticsParse.def

```diff
@@ -2158,6 +2158,10 @@ ERROR(init_accessor_is_not_on_property,none,
       "init accessors could only be associated with properties",
       ())
 
+ERROR(init_accessor_is_not_in_the_primary_declaration,none,
+      "init accessors could only be declared in the primary declaration",
```

Comment:
"Cannot be declared inside an extension" maybe?

Reply:
Reworded this, thanks!